NineChronicles.Headless.AccessControlCenter/Controllers/AccessControlServiceController.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using NineChronicles.Headless.AccessControlCenter.AccessControlService;
+using System.Linq;
+using Libplanet.Crypto;
+
+namespace NineChronicles.Headless.AccessControlCenter.Controllers
+{
+    [ApiController]
+    public class AccessControlServiceController : ControllerBase
+    {
+        private readonly IMutableAccessControlService _accessControlService;
+
+        public AccessControlServiceController(IMutableAccessControlService accessControlService)
+        {
+            _accessControlService = accessControlService;
+        }
+
+        [HttpGet("entries/{address}")]
+        public ActionResult<bool> IsAccessDenied(string address)
+        {
+            return _accessControlService.IsAccessDenied(new Address(address));
+        }
+
+        [HttpPost("entries/{address}/deny")]
+        public ActionResult DenyAccess(string address)
+        {
+            _accessControlService.DenyAccess(new Address(address));
+            return Ok();
+        }
```

Comment:
As a note, since there is no authentication, the operator should be careful about network policy.

Reply:
Yes, ACC is always on a private network. always...